### PR TITLE
Add TCK results Jakarta Authentication 3.0

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: Arjan Tijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+following is a summary of the TCK results for releases of Jakarta Authentication.
+
+# Eclipse GlassFish 7.0 Certification Summary
+
+- Product Name, Version and download URL (if applicable): <br/>
+  [Eclipse Glassfish 7.0.0-M4](https://github.com/eclipse-ee4j/glassfish/releases/download/7.0.0-M4/glassfish-7.0.0-M4.zip)
+- Specification Name, Version and download URL: <br/>
+  [Jakarta Authentication 3.0](https://jakarta.ee/specifications/authentication/3.0/)
+- TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta Authentication TCK, 3.0.0](https://download.eclipse.org/es/jakartaee10/staged/eftl/jakarta-authentication-tck-3.0.0-dist.zip), SHA-256: `f970fce5dd3883934ef4f1162c747c527f98cf88fcacc8d7af6ae4cf444c9ef8`
```
Comment (pzygielo) [inline docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md:15]: I'd expect TCK to be downloaded from `/ee4j/jakartaee10/...`.
Perhaps I missed something that justifies `/es`.

Comment (arjantijms) [reply docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md:15]: > I'd expect TCK to be downloaded from /ee4j/jakartaee10/....
> Perhaps I missed something that justifies /es.

It's the only location writable for the readily available security related CI. The other locations aren't writable, issues have been filed, but today is the deadline. We'll have to move it later and update this.

**Comment by arjantijms** [comment]
See also this one: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1239

**Review by dmatej — APPROVED** [review]

File: docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md (hunk 1 above)
Comment (arjantijms) [reply docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md]: I checked with @ivargrimstad and it appears that `/es` is absolutely fine. However, by then I had already moved it to `/ee4j/jakartaee10/...` and re-run the TCKs.

**Pull request merged**